wren-ui/src/apollo/server/adaptors/wrenAIAdaptor.ts

```diff
@@ -232,7 +232,7 @@ export class WrenAIAdaptor implements IWrenAIAdaptor {
       } else {
         return {
           status: WrenAIDeployStatusEnum.FAILED,
-          error: `WrenAI: Deploy wren AI failed or timeout, hash: ${hash}`,
+          error: `WrenAI: Deploy wren AI failed or timeout, hash: ${hash}, `,
```

Comment:
Why need a comma here?

Reply:
I was trying to print the error from the AI service
I'll remove it, thx